daemon/container.go

```diff
@@ -1441,6 +1442,18 @@ func (container *Container) startLogging() error {
 			return err
 		}
 		l = dl
+	case "gelf":
+		metadata := gelf.GelfMetadata{
```

Comment:
Yes make sense :)
Because every implemented driver need some metadata this information should be provided by the log driver api. When you implement a new driver there should be no need to do something outside of the driver api. But I guess this need some design decisions and refactoring of the log driver interface.